specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/schema/mqtt.kafka.schema.patch.json

```diff
@@ -109,11 +109,15 @@
                                                         "type": "array",
                                                         "items":
```

Comment:
@bmaidics, I updated some structures in this file that looked like they might be a copy-paste error. Can you please double-check to make sure I didn't change the intended structure?